src/pages/teams2.astro

```diff
@@ -0,0 +1,256 @@
+              <div
+                class="relative flex items-center justify-center w-11 h-11 rounded-full overflow-hidden"
+              >
+                <div
+                  class="absolute inset-0 bg-gradient-to-br from-blue-600 to-brand-pink"
+                >
+                </div>
+                <div
+                  class="relative w-9 h-9 flex items-center justify-center bg-gray-200 rounded-full border-4"
+                >
+                  <div class="relative">
+                    <div
+                      class="absolute w-1 h-6 bg-gradient-to-bl from-blue-600 to-brand-pink top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 rounded-md"
+                    >
+                    </div>
+                    <div
+                      class="absolute w-6 h-1 bg-gradient-to-bl from-blue-600 to-brand-pink top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 rounded-md"
+                    >
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <h1 class="text-2xl md:text-5xl font-semibold mt-2">
+            Livebook Teams features
+          </h1>
+          <div class="text-gray-600 text-xl mt-3">
+            A fast (and fun 😃) way to build internal tools.
+          </div>
+        </div>
+        <div id="build-with-elixir" class="mt-16">
+          <div id="build-with-elixir-title">
+            <div class="flex items-center gap-2">
+              <h2 class="text-xl md:text-4xl font-semibold">
+                Build with <span class="text-elixir">Elixir</span>
+              </h2>
+              <Image src={elixirLogo} alt="Elixir Logo" class="w-9 h-9" />
+            </div>
+            <div class="max-w-xl text-gray-700 text-lg mt-3 font-light">
+              No-code tools are good for non-technical teams, but as Elixir
+              developers, they constrain and even slow us down. With Livebook,
+              you get to build what you need using the Elixir tools and packages
+              you're familiar with.
+            </div>
+          </div>
+          <div
+            id="build-with-elixir-features"
+            class="mt-10 space-y-[75px] relative"
+          >
+            <div id="hex-libraries">
+              <div class="flex items-center justify-between">
+                <div>
+                  <div class="flex items-center">
+                    <StarIcon />
+                    <h3 class="text-xl font-semibold ml-4">
+                      Use Hex Libraries
+                    </h3>
+                  </div>
+                  <p class="text-md font-light max-w-sm ml-[52px] mt-2">
+                    Livebook is integrated with Hex and uses Mix.install to
+                    manage dependencies.
+                  </p>
+                </div>
+                <Image src={hexLibrary} alt="Hex Library" class="mr-28" />
```

Comment:
We should be able to export these as SVG from Figma I think, right? This way it's going to look more sharp :)